Project/Source/FileSystem/SceneImporter.cpp

```diff
@@ -36,20 +36,21 @@
 #define JSON_TAG_QUADTREE_ELEMENTS_PER_NODE "QuadtreeElementsPerNode"
 #define JSON_TAG_PARENT_ID "ParentId"
 
-static void ImportNode(const aiScene* assimpScene, const std::vector<Material>& materials, const aiNode* node, GameObject* parent, const float4x4& accumulatedTransform, std::vector<ResourceAnimation*> animations) {
+static GameObject* ImportNode(const aiScene* assimpScene, const std::vector<Material>& materials, const aiNode* node, GameObject* parent, const float4x4& accumulatedTransform) {
```

Comment:
¿Por qué cambiáis la lógica de esta función?

Reply:
Para usarlo en la linea 245 y tener referencia al gameobject que se creo . 
Si no se usara el añadir el component, tendriamos  que agregar las lineas 248-258 dentro del import node y  revisar contra el parent que no sea scene para verificar que solo se añada una vez el component  al padre de la jerarquia de ese gameobject